django/contrib/postgres/fields/hstore.py

```diff
@@ -51,6 +52,19 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(HStoreField, self).formfield(**defaults)
 
+    def get_prep_value(self, value):
+        value = super(HStoreField, self).get_prep_value(value)
+
+        if isinstance(value, dict):
```

Comment:
I'd consider changing this to `if value is not None` - I don't know whether other mappables can be handled lower down the stack? It's an edge case though and may be irrelevant as higher levels would have already rejected it.